workbench/settings.py

```diff
@@ -11,6 +11,23 @@
         'url_root': '/static/djpyfs'}
 
 DEBUG = True
+if os.environ.get('EXCLUDE_SAMPLE_XBLOCKS') == 'yes':
+    EXCLUDED_XBLOCKS = {
+        'allscopes_demo',
+        'attempts_scoreboard_demo',
+        'equality_demo',
+        'filethumbs',
+        'helloworld_demo',
+        'html_demo',
+        'problem_demo',
+        'sidebar_demo',
+        'slider_demo',
+        'textinput_demo',
+        'thumbs',
+        'view_counter_demo',
+    }
+else:
+    EXCLUDED_XBLOCKS = set()
```

Comment:
I think I would have made the list of names of excluded XBlocks be the value of the environment variable, to avoid this duplicated list of xblock names, and to give you flexibility into the future if more samples are added in the future.

But I won't bike-shed :)